lib/lago_eu_vat/lago_eu_vat/rate.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module LagoEuVat
+  class Rate
+    def initialize
+      file_path = Rails.root.join('lib', 'lago_eu_vat', 'lago_eu_vat', 'eu_vat_rates.json')
+      json_file = File.read(file_path)
+      @json_countries_rates = JSON.parse(json_file)['items']
+    end
+
+    def countries_code
+      json_countries_rates.map { |country_code, _| country_code }
+    end
+
+    def country_rates(country_code:)
+      # NOTE: country rates are ordered by date, so we select the most recent applicable
+      json_countries_rates[country_code].select do |period|
+        Time.zone.now >= DateTime.parse(period['effective_from'])
+      end&.first.fetch('rates')
```

Comment:
Lint/SafeNavigationChain: Do not chain ordinary method call after safe navigation operator.